exercises/movr/rides/data/rides_database.sql

```diff
@@ -1,15 +1,15 @@
 CREATE DATABASE movr_rides;
 
 CREATE TABLE movr_rides.rides (
-	id UUID NOT NULL DEFAULT gen_random_uuid(),
+	id UUID PRIMARY KEY NOT NULL DEFAULT gen_random_uuid(),
```

Comment:
PRIMARY KEY includes the properties of NOT NULL and UNIQUE. See the docs PK page Details: https://www.cockroachlabs.com/docs/stable/primary-key.html#details

That means you don't need NOT NULL in a declaration that uses PRIMARY KEY. 

Please remove the NOT NULL in all the id cols with PRIMARY KEY. Thanks!

Reply:
removed

Reply:
Thx @kikiya !